src/System.Management.Automation/commands/utility/FormatAndOutput/common/DisplayDatabase/displayDescriptionData.cs

```diff
@@ -713,6 +718,11 @@ public abstract class PSControl
         /// </summary>
         public bool OutOfBand { get; set; }
 
+        /// <summary>
+        /// Set explicit limit for Width. Default width is a console host width.
+        /// </summary>
+        public int? Width { get; set; }
+
```

Comment:
We still don't have tests for this area. 😕 
I added `Width` in `RehydratePSControl`. I don't changed 'CompatibleWithOldPowerShell'.
I still don't understand how serialization works in depth so I rely to your experience.